experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.settings.ts

```diff
@@ -0,0 +1,86 @@
+import { radioButtonName, IRadioButtonType } from './RadioButton.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { ViewProps } from 'react-native';
+
+export const settings: IComposeSettings<IRadioButtonType> = [
+  {
+    root: {
+      accessible: true,
+      acceptsKeyboardFocus: true,
+      accessibilityRole: 'radio',
+      style: {
+        display: 'flex',
+        alignItems: 'flex-start',
+        flexDirection: 'row',
+        alignSelf: 'flex-start',
+        minHeight: 20,
+        marginTop: 0,
+        position: 'relative'
+      }
+    } as ViewProps,
+    button: {
+      style: {
+        backgroundColor: 'transparent',
+        width: 20,
+        height: 20,
+        top: 0,
+        left: 0,
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderColor: 'black',
+        borderRadius: 50,
```

Comment:
50% which creates a circle.

Reply:
then it should be `50%`

Reply:
Both 50% and '50%' cause an error. 50 seems to work.

Reply:
This is weird. @lenahong Is '50%' supposed to work? 50 shouldn't work. We need to create a bug for this.

Reply:
I just checked back on my borderRadius work and we decided not to support percentage for borderRadius because React Native doesn't. The reason that 50 worked is because any number beyond the maxinumum borderRadius is cut off at what the 50% of the width/height would be, in this case 10. If we want to eventually expose the width/height token for customizing radioButton size, I suggest creating a helper function that does the 50% calculation for the token processor.

Reply:
I agree with Lena's suggestion. That is also the approach that I took for the Persona control. Basically, you calculate the width or height and divide by 2 to set the border radius. 

Setting it to 50 here is not the right solution.

Reply:
I agree this is not the right solution. However, tokens were in the plan for V2. Currently for V1, we don't support tokens. Do you think we can submit as is for V1, and then this sprint I'll work on getting the tokens working?

Reply:
Will be addressed in V2, opening a task for it. In the meantime I added constants at the top of the settings file to communicate intent better